src/interfaces/booking.interfaces.ts

```diff
@@ -14,3 +14,15 @@ export interface DatePickerInterface {
   selectedDate?: Date | null
   setSelectedDate?: (date: Date) => void 
 }
+
```

Comment:
If you need to comment the example of data pass to this Interface you can write above Interface to show the example.It's better that write in field

Reply:
Luv your observation point !